### PR TITLE
Update reasoning for disabled tests in SpecialCharsTest

### PR DESCRIPTION
**Pull request by rsvoboda**
Update reasoning for disabled tests in SpecialCharsTest

### CODE REVIEW

File: testsuite/src/it/java/io/quarkus/ts/startstop/SpecialCharsTest.java [hunk 1]
```diff
@@ -183,14 +183,14 @@ public void specialJVM(TestInfo testInfo) throws IOException, InterruptedExcepti
...
-    @DisabledOnOs({OS.WINDOWS}) // https://github.com/quarkusio/quarkus/issues/9707
+    @DisabledOnOs(value = {OS.WINDOWS}, disabledReason = "https://github.com/quarkusio/quarkus/issues/42248")
     public void specialDEV(TestInfo testInfo) throws IOException, InterruptedException {
         testRuntime(testInfo, Apps.JAKARTA_REST_MINIMAL, MvnCmds.DEV, ",;~!@#$%^&()");
     }
 
     @Test
     @Tag("native")
-    @DisabledOnOs({OS.WINDOWS}) // https://github.com/quarkusio/quarkus/issues/9707
+    @DisabledOnOs(value = {OS.WINDOWS}, disabledReason = "https://github.com/quarkusio/quarkus/issues/42248")
```
Comment (jedla97) [inline testsuite/src/it/java/io/quarkus/ts/startstop/SpecialCharsTest.java:193]: As this is native test shouldn't be reason `"Native on Windows is not supported"`. On the next line we disabling the whole test with same issue. I would expect this test to work when the issue is fixed or am I missing something why this should not work on windows?

File: testsuite/src/it/java/io/quarkus/ts/startstop/SpecialCharsTest.java [hunk 2]
```diff
@@ -183,14 +183,14 @@ public void specialJVM(TestInfo testInfo) throws IOException, InterruptedExcepti
     }
 
     @Test
-    @DisabledOnOs({OS.WINDOWS}) // https://github.com/quarkusio/quarkus/issues/9707
+    @DisabledOnOs(value = {OS.WINDOWS}, disabledReason = "https://github.com/quarkusio/quarkus/issues/42248")
```
Comment (jedla97) [inline testsuite/src/it/java/io/quarkus/ts/startstop/SpecialCharsTest.java:186]: Why this should be disabled only on Windows? The new issue is OS indipendent so I would be expecting that this either work on all platform or nowhere.

**Review by jedla97 — CHANGES_REQUESTED** [review]
Thanks for updating the reasons I just have two comments about disabling the special characters tests (`,;~!@#$%^&()`).

File: testsuite/src/it/java/io/quarkus/ts/startstop/SpecialCharsTest.java (hunk 2 above)
Comment (rsvoboda) [reply testsuite/src/it/java/io/quarkus/ts/startstop/SpecialCharsTest.java:186]: Gosh, the right issue is https://github.com/quarkusio/quarkus/issues/42942

**Comment by rsvoboda** [comment]
I will update the PR in ~1 hour

**Review by jedla97 — APPROVED** [review]
Thanks for update

**Pull request merged**